Assets/AltTester/Runtime/UI/AltDialog.cs

```diff
@@ -314,13 +315,24 @@ private void StopClient()
                 _liveUpdateCommunication = null;
             }
         }
-        private void OnDisconnect()
+
+        private void OnDisconnect(int code, string reason)
         {
-            _updateQueue.ScheduleResponse(() =>
+            // All custom close codes must be between 4000 - 4999.
+            if (code > 4000)
             {
-                StopClient();
-            });
-
+                _updateQueue.ScheduleResponse(() =>
+                {
+                    SetMessage(reason, ERROR_COLOR, true);
```

Comment:
Shouldn't we need to stop the clients if there is an error?

Reply:
If we stop the client, will set the variables to `null` and in `Update` we will start them again. This way the error is displayed and when the user hits the restart button the clients will be stopped and started again.